Core/GDCore/Project/VariablesContainer.h

```diff
@@ -33,7 +33,9 @@ class GD_CORE_API VariablesContainer {
       Global,
       Scene,
       Object,
-      Local
+      Local,
+      ExtensionGlobal,
+      Extension

```

Comment:
As discussed, maybe 
```suggestion
      ExtensionScene

```

Reply:
(someone could say that there is no reason "Extension" would mean something tied to the scene, the default "Extension" could as well mean "global variables" that live forever because the extension lives forever when the game runs)

Reply:
I agree but I used a different enum value because I was afraid of a confusion in the generator. Maybe we can try to refactor later?

Reply:
You mean a confusion if you rename Extension to ExtensionScene?

That should be fairly safe now, the compiler will let you know all the changes to do?

Reply:
No, I meant using existing `Scene` and `Global` + the scope as there is no "any" variable that can be both `Global` and `ExtensionGlobal` but I'm not sure if it would be better or not.